package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "emulsify-gulp",
   "description": "A collection of useful gulp tasks.",
-  "version": "1.1.1",
+  "version": "2.0.0",
```

Comment:
This should probably be `2.0.1` unless you use `npm version patch` to create your tags/bump versions, in which case this if fine. lmk if this needs explanation.

Reply:
The team is manually tagging currently so this is not an issue. `npm version` can be used in the future.